x/wasmplus/keeper/msg_server_integration_test.go

```diff
@@ -67,25 +96,27 @@ func TestStoreAndInstantiateContract(t *testing.T) {
 			require.NoError(t, wasmApp.AppCodec().Unmarshal(rsp.Data, &storeAndInstantiateResponse))
 
 			// check event
-			events := rsp.Events
-			assert.Equal(t, 3, len(events))
-			assert.Equal(t, "store_code", events[0].Type)
-			assert.Equal(t, 2, len(events[0].Attributes))
-			assert.Equal(t, "code_checksum", string(events[0].Attributes[0].Key))
-			assert.Equal(t, "code_id", string(events[0].Attributes[1].Key))
-			assert.Equal(t, "1", string(events[0].Attributes[1].Value))
-			assert.Equal(t, "message", events[1].Type)
-			assert.Equal(t, 2, len(events[1].Attributes))
-			assert.Equal(t, "module", string(events[1].Attributes[0].Key))
-			assert.Equal(t, "wasm", string(events[1].Attributes[0].Value))
-			assert.Equal(t, "sender", string(events[1].Attributes[1].Key))
-			assert.Equal(t, "instantiate", events[2].Type)
-			assert.Equal(t, "_contract_address", string(events[2].Attributes[0].Key))
-			assert.Equal(t, storeAndInstantiateResponse.Address, string(events[2].Attributes[0].Value))
-			assert.Equal(t, "code_id", string(events[2].Attributes[1].Key))
-			assert.Equal(t, "1", string(events[2].Attributes[1].Value))
+			assert.Equal(t, spec.expEvents, rsp.Events)
 
 			require.NoError(t, err)
 		})
 	}
 }
+
+// This function is utilized to generate the msg event for event checking in integration tests
+// It will be deleted in release/v0.1.x
+func createMsgEvent(sender sdk.AccAddress) abci.Event {
+	return abci.Event{
+		Type: "message",
+		Attributes: []abci.EventAttribute{
+			{
+				Key:   []byte("module"),
+				Value: []byte("wasm"),
+			},
```

Comment:
If you want to test all fields, please add the Index field

Reply:
Thanks. That's right.